src/DeterministicVI.jl

```diff
@@ -34,13 +41,18 @@ function ElboArgs{NumType <: Number}(
             vp::VariationalParams{NumType},
             tile_source_map::Vector{Matrix{Vector{Int}}},
             patches::Matrix{SkyPatch},
-            active_sources::Vector{Int})
+            active_sources::Vector{Int},
+            psf_K::Int,
```

Comment:
Do we need a default value for `psf_K`? If so, would you make `psf_K` a keyword argument with a default value, rather than having a constant named `default_psfK`?